api/observability/v1/output_types.go

```diff
@@ -613,6 +619,46 @@ type LokiStack struct {
 	LabelKeys []string `json:"labelKeys,omitempty"`
 }
 
+// Loki provides optional extra properties for `type: loki`
+type Loki struct {
+	// Authentication sets credentials for authenticating the requests.
+	//
+	// +kubebuilder:validation:Optional
+	Authentication *HTTPAuthentication `json:"authentication,omitempty"`
+
+	// Tuning specs tuning for the output
+	//
+	// +kubebuilder:validation:Optional
+	// +nullable
+	Tuning *LokiTuningSpec `json:"tuning,omitempty"`
+
+	URLSpec `json:",inline"`
+
+	// LabelKeys is a list of log record keys that will be used as Loki labels with the corresponding log record value.
+	//
+	// If LabelKeys is not set, the default keys are `[log_type, kubernetes.namespace_name, kubernetes.pod_name, kubernetes_host]`
+	//
+	// Note: Loki label names must match the regular expression "[a-zA-Z_:][a-zA-Z0-9_:]*"
+	// Log record keys may contain characters like "." and "/" that are not allowed in Loki labels.
+	// Log record keys are translated to Loki labels by replacing any illegal characters with '_'.
+	// For example the default log record keys translate to these Loki labels: `log_type`, `kubernetes_namespace_name`, `kubernetes_pod_name`, `kubernetes_host`
+	//
+	// Note: the set of labels should be small, Loki imposes limits on the size and number of labels allowed.
+	// See https://grafana.com/docs/loki/latest/configuration/#limits_config for more.
+	// Loki queries can also query based on any log record field (not just labels) using query filters.
+	//
+	// +kubebuilder:validation:Optional
+	LabelKeys []string `json:"labelKeys,omitempty"`
+
+	// TenantKey is a meta-data key field to use as the TenantID,
+	// For example: 'TenantKey: kubernetes.namespace_name` will use the kubernetes
+	// namespace as the tenant ID.
+	//
+	// +kubebuilder:validation:Required
+	// +kubebuilder:default:=.log_type
```

Comment:
I'm fairly certain we still require the `{{}}` since that is the indicator of templating for vector.  I anticipate this is the indicator as well to say its a dynamic versus static value.  We likely need to add more validation to this and other places that have a "tenant" value (e.g. index)